cal/static/jsx/query.jsx

```diff
@@ -1,10 +1,39 @@
 var RSearch = React.createClass({
+    onclick: function() {
+        console.log("Clicked!");
+        tag = $("input")[0]
+        url = "/search/" + tag.value
+        var setGlobalState = this.props.setGlobalState;
```

Comment:
Do we need `setGlobalState`? Why don't we just use `this.props.setGlobalState` everywhere? (If it makes the lines too long, that's fine, but otherwise, I'd rather we avoid unnecessary variables).